### DISCUSSION TITLE
Sensor type

### DISCUSSION THREAD

**Discussion by yinma-tech** [UmerCodez/SensorServer#36] [Q&A]
First, thank you for providing this useful app, if you can issue a new version that contain the raw sensor data, I would greatly appreciate it . (ps, i think the sensor data is filtered data)

**Comment by UmerCodez** [comment]
The sensor data reported by Android's Sensor Framework is not filtered data 
>SensorEvent
The system uses this class to create a sensor event object, which provides information about a sensor event. A sensor event object includes the following information: **the raw sensor data**, the type of sensor that generated the event, the accuracy of the data, and the timestamp for the event.

Source : https://developer.android.com/guide/topics/sensors/sensors_overview

https://github.com/umer0586/SensorServer/blob/21f09fa037a5aa9ab83e3e0bdf486ebcb25daa84/app/src/main/java/github/umer0586/sensorserver/websocketserver/SensorWebSocketServer.kt#L561-L603

From above code you can see that no filtering is performed before sending data to over Websocket.
